index.html

```diff
@@ -87,6 +87,7 @@
         openmct.install(openmct.plugins.Filters(['table', 'telemetry.plot.overlay']));
         openmct.install(openmct.plugins.ObjectMigration());
         openmct.install(openmct.plugins.GoToOriginalAction());
+        openmct.install(openmct.plugins.ClearData(['table', 'telemetry.plot.overlay', 'telemetry.plot.stacked']));
```

Comment:
On re-reading this, I think that this action should be installed by default and not in index.html. We should keep index.html for plugins that users are likely to want to customize, and make sure it's not growing in an unmanageable way. I think this also applies to some of the other plugins installed in `index.html` right now. We should make "Go To Original", "LADTable", "FolderView", "Tabs", and "FlexibleLayout" as this stuff is basically all stuff that the platform provides by default. Thoughts?